src/datasets/features/audio.py

```diff
@@ -131,6 +148,8 @@ def decode_example(self, value: dict, token_per_repo_id=None) -> dict:
                 array, sampling_rate = self._decode_non_mp3_path_like(
                     path, "opus", token_per_repo_id=token_per_repo_id
                 )
+        elif path is not None and path.endswith("pcm"):
+            sampling_rate, array = wavfile.read(file)
```

Comment:
It's not possible to decode a pcm file, since we don't know the sampling rate, you can raise an error here:
```suggestion
        elif path is not None and path.endswith("pcm"):
            raise ValueError(f"Unable to decode PCM file at {path} since the sampling rate is unknown.")
```

Reply:
how about don`t compare path? that is not necessary anymore i think.
so i will commit again